mne/viz/ica.py

```diff
@@ -327,7 +365,22 @@ def plot_ica_scores(ica, scores, exclude=None, axhline=None,
     else:
         axes = [axes]
     plt.suptitle(title)
-    for this_scores, ax in zip(scores, axes):
+
+    if labels == 'ecg':
+        labels = [l for l in ica.labels_ if l.startswith('ecg')]
+    elif labels == 'eog':
+        labels = [l for l in ica.labels_ if l.startswith('eog/')]
```

Comment:
I'm fine using a different separator, e.g. a dollar or semicolon, etc. and protecting it (throwing erorrs) if you think that's a good idea.